docs/tutorials/faq/main.md

```diff
@@ -196,6 +197,7 @@ and send out on a different transport. For example, receive TLS on IPv6
 and send out UDP on IPv4.
 
 ### Can Kamailio be used to call from web browsers (webrtc) to classic
+
 SIP phones?
```

Comment:
This was supposed to be a single line with the question, so it should be merged on previous one, than adding an empty line in between.